sqs/src/main/scala/akka/stream/alpakka/sqs/SqsSourceSettings.scala

```diff
@@ -211,24 +211,6 @@ case object MessageDeduplicationId extends AttributeName("MessageDeduplicationId
 case object MessageGroupId extends AttributeName("MessageGroupId")
 case object SequenceNumber extends AttributeName("SequenceNumber")
 
-case object Policy extends AttributeName("Policy")
```

Comment:
These attributes are available in `sqs.model.QueueAttributeName`. Can you find a way to keep these classes as deprecated in Alpakka for the time being? Eg. make them use the the enum.

Reply:
It would be possible to classify these queue-attributes as `QueueAttributeName`'s (next to the already existing `AttributeName`'s). They would not be deleted nor deprecated in that case, they 'just' wouldn't be used:

```scala
sealed abstract class QueueAttributeName(val name: String)

case object Policy extends QueueAttributeName("Policy")
case object VisibilityTimeout extends QueueAttributeName("VisibilityTimeout")
case object MaximumMessageSize extends QueueAttributeName("MaximumMessageSize")
case object MessageRetentionPeriod extends QueueAttributeName("MessageRetentionPeriod")
case object ApproximateNumberOfMessages extends QueueAttributeName("ApproximateNumberOfMessages")
case object ApproximateNumberOfMessagesNotVisible extends QueueAttributeName("ApproximateNumberOfMessagesNotVisible")
case object CreatedTimestamp extends QueueAttributeName("CreatedTimestamp")
case object LastModifiedTimestamp extends QueueAttributeName("LastModifiedTimestamp")
case object QueueArn extends QueueAttributeName("QueueArn")
case object ApproximateNumberOfMessagesDelayed extends QueueAttributeName("ApproximateNumberOfMessagesDelayed")
case object DelaySeconds extends QueueAttributeName("DelaySeconds")
case object ReceiveMessageWaitTimeSeconds extends QueueAttributeName("ReceiveMessageWaitTimeSeconds")
case object RedrivePolicy extends QueueAttributeName("RedrivePolicy")
case object FifoQueue extends QueueAttributeName("FifoQueue")
case object ContentBasedDeduplication extends QueueAttributeName("ContentBasedDeduplication")
case object KmsMasterKeyId extends QueueAttributeName("KmsMasterKeyId")
case object KmsDataKeyReusePeriodSeconds extends QueueAttributeName("KmsDataKeyReusePeriodSeconds")
```

Reply:
I was not sure if I get your point with "using the enum", but if get it right it would look like this:

```scala
import software.amazon.awssdk.services.sqs.model
sealed abstract class QueueAttributeName(val name: model.QueueAttributeName)

case object Policy extends QueueAttributeName(model.QueueAttributeName.POLICY)
case object VisibilityTimeout extends QueueAttributeName(model.QueueAttributeName.VISIBILITY_TIMEOUT)
case object MaximumMessageSize extends QueueAttributeName(model.QueueAttributeName.MAXIMUM_MESSAGE_SIZE)
case object MessageRetentionPeriod extends QueueAttributeName(model.QueueAttributeName.MESSAGE_RETENTION_PERIOD)
case object ApproximateNumberOfMessages extends QueueAttributeName(model.QueueAttributeName.APPROXIMATE_NUMBER_OF_MESSAGES)
case object ApproximateNumberOfMessagesNotVisible extends QueueAttributeName(model.QueueAttributeName.APPROXIMATE_NUMBER_OF_MESSAGES_NOT_VISIBLE)
case object CreatedTimestamp extends QueueAttributeName(model.QueueAttributeName.CREATED_TIMESTAMP)
case object LastModifiedTimestamp extends QueueAttributeName(model.QueueAttributeName.LAST_MODIFIED_TIMESTAMP)
case object QueueArn extends QueueAttributeName(model.QueueAttributeName.QUEUE_ARN)
case object ApproximateNumberOfMessagesDelayed extends QueueAttributeName(model.QueueAttributeName.APPROXIMATE_NUMBER_OF_MESSAGES_DELAYED)
case object DelaySeconds extends QueueAttributeName(model.QueueAttributeName.DELAY_SECONDS)
case object ReceiveMessageWaitTimeSeconds extends QueueAttributeName(model.QueueAttributeName.RECEIVE_MESSAGE_WAIT_TIME_SECONDS)
case object RedrivePolicy extends QueueAttributeName(model.QueueAttributeName.REDRIVE_POLICY)
case object FifoQueue extends QueueAttributeName(model.QueueAttributeName.FIFO_QUEUE)
case object ContentBasedDeduplication extends QueueAttributeName(model.QueueAttributeName.CONTENT_BASED_DEDUPLICATION)
case object KmsMasterKeyId extends QueueAttributeName(model.QueueAttributeName.KMS_MASTER_KEY_ID)
case object KmsDataKeyReusePeriodSeconds extends QueueAttributeName(model.QueueAttributeName.KMS_DATA_KEY_REUSE_PERIOD_SECONDS)
```

Using the same approach for the `AttributeName` could be a little bit tricky due to aws/aws-sdk-java#2061. There is already an enum `sqs.model.MessageSystemAttributeName` modelling the message attributes but this one is currently missing the `All` option:

```scala
sealed abstract class AttributeName(val name: MessageSystemAttributeName)

case object All extends AttributeName(???)
case object ApproximateFirstReceiveTimestamp extends AttributeName(MessageSystemAttributeName.APPROXIMATE_FIRST_RECEIVE_TIMESTAMP)
```

Reply:
What about defining them as 
```scala
sealed abstract class MessageSystemAttributeName(_name: String) extends AttributeName(_name) {
  protected def this(msname: model.MessageSystemAttributeName) {
    this(msname.toString)
  }
}
sealed abstract class QueueAttributeName(qaname: model.QueueAttributeName) extends AttributeName(qaname.toString)
```

Reply:
Applied your suggestion, can you please re-review?